libs/execution_context/ebpf_native.c

```diff
@@ -1306,7 +1300,8 @@ ebpf_native_unload(_In_ const GUID* module_id)
     ebpf_native_module_t* module = NULL;
     wchar_t* service_name = NULL;
     size_t service_name_length;
-    bool unload_module = false;
+    bool queue_workitem = false;
```

Comment:
```suggestion
    bool queue_work_item = false;
```
nit: consistency with line 1304